src/api/spec/factories/project.rb

```diff
@@ -27,6 +27,12 @@
       project.write_to_backend
     end
 
+    trait :as_submission_source do
+      after(:create) do |project, _evaluator|
```

Comment:
[nitpick] `_` instead `_evaluator`

Reply:
yeah, but often enough I still need it - and then it's easier to remove the _